.github/workflows/update-nextcloud-ocp.yml

```diff
@@ -17,12 +17,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branches: ["master", "stable27", "stable26", "stable25", "stable24"]
+        branches: ["main", "master", "stable26", "stable25", "stable24"]
```

Comment:
missing 27?

Reply:
wut

Reply:
Maybe I ran too fast :thinking:

Reply:
Ah no, this is a different workflow

Reply:
https://github.com/nextcloud/.github/pull/161